modules/apps/commerce/commerce-product-service/src/main/java/com/liferay/commerce/product/service/impl/CPDefinitionLocalServiceImpl.java

```diff
@@ -2764,6 +2766,7 @@ private CPDefinitionLocalization _addCPDefinitionLocalizedFields(
 			cpDefinitionLocalization.setMetaTitle(metaTitle);
 			cpDefinitionLocalization.setMetaDescription(metaDescription);
 			cpDefinitionLocalization.setMetaKeywords(metaKeywords);
+			cpDefinitionLocalization.setCProductId(cProductId);
```

Comment:
I think we should probably sort this.
Maybe like,
`cpDefinitionLocalization.setCompanyId(companyId);
			cpDefinitionLocalization.setCPDefinitionId(cpDefinitionId);
cpDefinitionLocalization.setCProductId(cProductId);`